code/modules/mob/living/carbon/human/species_types/oozelings.dm

```diff
@@ -104,11 +104,15 @@
 /datum/species/oozeling/proc/Cannibalize_Body(mob/living/carbon/human/H)
 	var/list/limbs_to_consume = list(BODY_ZONE_R_ARM, BODY_ZONE_L_ARM, BODY_ZONE_R_LEG, BODY_ZONE_L_LEG) - H.get_missing_limbs()
 	var/obj/item/bodypart/consumed_limb
+	var/L
```

Comment:
Declaring the variable inside the for is the norm

Reply:
Like for (var/L ......)

Reply:
Changed it to declare the var inside the for() loop. Did it the other way as that's how the DM reference guide shows it (probably for ease of reading/teaching).